src/models/clients.models.ts

```diff
@@ -6,6 +6,15 @@ export function fetchClients() {
   });
 }
 
+export function getClient(data) {
+  const gov_id = [data.gov_id];
+  return db
+    .query("SELECT * FROM clients WHERE gov_id = $1", gov_id)
+    .then((client) => {
+      return client.rows;
```

Comment:
hmm instead of returning an array just do `client.rows[0]`, would be easier for when you use it in the controller